packages/frontend/src/pages/Models.svelte

```diff
@@ -145,6 +147,15 @@ async function importModel() {
               {/if}
             </Route>
 
+            <!-- Imported models -->
+            <Route path="/imported">
+              {#if importedModels.length > 0}
+                <Table kind="model" data="{importedModels}" columns="{columns}" row="{row}"></Table>
+              {:else}
+                <div role="status">There are no models yet</div>
```

Comment:
I'm not sure which expression is correct, but this one is different from the one in other tabs ('There is no model yet')